ScoutSuite/frontend/src/utils/Partials/index.js

```diff
@@ -41,6 +50,35 @@ export const valueOrNone = value => {
  */
 export const concatPaths = (pathA, pathB) => pathA.length > 0 ? `${pathA}.${pathB}` : pathB;
 
+/**
+ * Format Date
+ * @param time
+ * @returns {string}
+ */
+export const formatDate = time => {
```

Comment:
I saw you added a similar function in your PR, I added the exact code from the old front-end. I'm not sure if it's worth keeping or if we should just use yours.

Reply:
Let's use this one :)